lib/src/widgets/slider_picker.dart

```diff
@@ -9,10 +9,17 @@ class SliderPicker extends StatefulWidget {
     this.max = 1.0,
     this.colors,
     this.child,
+    BorderRadius? borderRadius,
+    this.border,
+    double? height,
     Key? key,
-  })  : assert(value >= min && value <= max),
+  })  : height = height ?? 40.0,
```

Comment:
Same as above. These values can be used as the default value, rather than constructor syntaxes.